elastic/kibana
# [Index Management] Adopt data stream API changes

Multi-part towards https://github.com/elastic/kibana/issues/71670

This PR updates the data streams API integration test and fixes the composable templates creation flow.

### How to test
Verify the composable index template creation flow works as expected when enabling the data streams toggle.
